phpstan.neon.dist

```diff
@@ -14,3 +16,5 @@ parameters:
         - '#Call to an undefined method TModel of Illuminate\\Database\\Eloquent\\Model#'
         - '#Call to an undefined method Illuminate\\Database\\Eloquent\\Model::#'
         - '#Call to an undefined method Illuminate\\Database\\Eloquent\\Builder#'
+        - identifier: expr.resultUnused
+          path: tests
```

Comment:
It's generally not a good idea to have phpstan analyze your tests I would recommend only analyzing the src directory and removing the phpunit plugins

Reply:
I remember reading something by the creator of PHPStan recommending to run it on the tests folder too, although I can't find it back using Google now

Reply:
I always thought that analyzing tests makes sense to see if the package's types also work in combination with actual applications (like when the user defines their own relationships with the package's methods).

Reply:
Excluding tests might reduce PHPStan errors in this project, but it might hide errors that are faced in user land which are tested in this package. So I actually think we should try and solve these errors, instead of hiding them

Reply:
Generally you test the phpstan stuff with type tests and integration tests (you can see Laravel framework and Larastan for examples) instead of analyzing all tests

Reply:
Thanks @calebdw. I'll remove the `tests` path for now and look into type tests.